Makefile

```diff
@@ -9,6 +9,8 @@ MODULE_big = das_fdw
 # Versions of the protobuf
 RAW_VERSION = v0.39.0
 DAS_VERSION = v0.1.2
+PG_CFLAGS = -O0 -g
+PG_LIBS = -L$(shell $(PG_CONFIG) --pkglibdir) -lhstore
```

Comment:
Relates to an attempt to link with the hstore lib. I believe eventually something wasn't working and I